cangen.c

```diff
@@ -166,6 +171,7 @@ int main(int argc, char **argv)
 	struct ifreq ifr;
 
 	struct timespec ts;
+	struct timespec tiny_ts;
```

Comment:
What's the purpose of `tiny_ts`?

Reply:
I was having problems with the program almost immediately filling by TX buffer and throwing an error locking up. Having a very small sleep between tx messages in each burst solved the problem.

Reply:
Call the programm with a polltimeout `-p 1000`.

Reply:
ok I see. makes sense.